### PR TITLE
Release 1.0.1 Change-log updated

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### ISSUE TYPE

 
- Docs Pull Request
 
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/CHANGELOG.rst [hunk 1]
```diff
@@ -5,10 +5,13 @@ Ansible Network Collection Release Notes
...
 
-v1.0.0
+v1.0.1
 ======
 
 Release Summary
 ---------------
 
-- Released 1.0.0 with updated changelog.
+- Rereleased 1.0.0 with updated README.md and regenerated documentation.
```
Comment (trishnaguha) [inline changelogs/CHANGELOG.rst:14]: Just "README documentation update" should be enough.  "Rerelease" also sounds like we are overwriting 1.0.0.

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-    changes:
-      release_summary:
-      - Released 1.0.0 with updated changelog.
     fragments:
     - test.yaml
     release_date: '2021-05-18'
+  1.0.1:
+    changes:
+      release_summary:
+      - Rereleased 1.0.0 with updated README.md and regenerated documentation.
```
Comment (trishnaguha) [inline changelogs/changelog.yaml:10]: Same for this

**Review by trishnaguha — CHANGES_REQUESTED** [review]

**Review by Qalthos — COMMENTED** [review]
I agree with Trishna, but no complaints otherwise

File: changelogs/CHANGELOG.rst (hunk 1 above)
Comment (rohitthakur2590) [reply changelogs/CHANGELOG.rst]: sounds good. Updated !

**Review by trishnaguha — APPROVED** [review]

**Pull request merged**